samples/subsys/usb/uvc/sample.yaml

```diff
@@ -0,0 +1,24 @@
+sample:
+  name: USB Video sample
+tests:
+  sample.subsys.usb.uvc:
+    depends_on:
+      - usbd
+    tags: usb
+    integration_platforms:
+      - nrf52840dk/nrf52840
+      - nrf54h20dk/nrf54h20/cpuapp
+      - frdm_k64f
+      - stm32f723e_disco
+      - nucleo_f413zh
+      - mimxrt685_evk/mimxrt685s/cm33
+      - mimxrt1060_evk/mimxrt1062/qspi
+    extra_args:
+      - EXTRA_DTC_OVERLAY_FILE="video-emul.overlay"
+  sample.subsys.usb.uvc.camera:
+    depends_on:
+      - usbd
+    tags: usb video
+    filter: dt_chosen_enabled("zephyr,camera")
+    integration_platforms:
+      - arduino_nicla_vision/stm32h747xx/m7
```

Comment:
I see that it depends on video-dcmi.overlay. I thought the source was abstracted by the video driver/subsystem. If there is no generic way to get the video source, then you cannot use `filter: dt_chosen_enabled("zephyr,camera")` and integration_platform because it would fail on other platforms.